test/spec.index.js

```diff
@@ -250,6 +250,57 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('by default, assumes the field isn\'t required', function () {
```

Comment:
There's definitely a mismatch.How would you override a 'required' validator? Following an approach that requires the minimum configuration makes the most sense to me.